charts/apisix-ingress-controller/templates/rbac.yaml

```diff
@@ -14,11 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: {{ .Release.Name }}-serviceaccount
----
+{{- if .Values.rbac.create }}
```

Comment:
``{{-`` makes the first line inside the if block joint with the last line, which will cause `apiVersion` commented. You should use `{{` here.

Reply:
`{{-` is right, `{{` will keep a blank line.
```
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#

apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
```

Reply:
It's OK to keep the blank line, but it's a syntax error if the `apiVersion` field is commented.

See https://github.com/helm/helm/issues/9225 for the details.

Reply:
`{{- if .Values.rbac.create }}` and `{{- if .Values.rbac.create -}}` are different.

`{{- if .Values.rbac.create -}}` will cause `apiVersion` commented

Reply:
OK, good to know that.